docs/UserGuide.md

```diff
@@ -189,7 +195,7 @@ Format: `odel INDEX`
 Example:
 * `odel 1` Deletes the 1st room in the room list.
 
-### Allocate resident to room 
+### Allocate resident to room: `alloc`
 Allocates an existing resident to an existing room. 
 
 Format: `alloc n/NAME r/ROOM_NO`
```

Comment:
Could you add a clarification here whether the name match needs to be an exact match or not? And does it have to be case sensitive? Would be a useful caveat to have for users of the system who may not be 100% familiar with the implementation of the function.